db/migrations/000006_wordvault_decks.up.sql

```diff
@@ -0,0 +1,34 @@
+BEGIN;
+
+-- Schema changes
+CREATE TABLE wordvault_decks (
+    id BIGSERIAL PRIMARY KEY,
+    user_id BIGINT NOT NULL,
+    lexicon_name TEXT NOT NULL,
+    fsrs_params_override JSONB DEFAULT null,
```

Comment:
Not currently exposed at the API layer but will be a feature to add pretty shortly after decks are shipped